src/api-reference/expense/expense-report/v4.workflows.markdown

```diff
@@ -101,64 +121,222 @@ PATCH operations in Expense Reports v4 conform to the JSON Merge Patch specifica
 
 * [Approve/SendBack/Recall Report Response](#patch-approve-sendback-recall-report-response-schema)
 
-### Example
+## View Cost Objects for a specific Report
+
+This endpoint retrieves all cost objects for a report that the calling approver or their delegators is able to approve.
+If the workflow setting "Filter expenses to those that are applicable to Cost Object" is set to false, then this
+endpoint will return all cost objects related to the report regardless of if the approver has permission to approve
+them.
+
+### <a name="access-token-usage"></a>Access Token Usage
 
-#### <a name="patch-approve-sendback-recall-report-request-schema"></a>Sample Requests (short and long URL)
+This API supports both company level and user level access tokens.
+
+### Scopes
+
+`expense.report.read` - Refer to [Scope Usage](#scope-usage) for full details.
+
+`expense.report.readwrite` - Refer to [Scope Usage](#scope-usage) for full details.
+
+### Parameters
+
+| Name       | Type     | Format | Description                                                                                                                                         |
+|------------|----------|--------|-----------------------------------------------------------------------------------------------------------------------------------------------------|
+| `userID`   | `string` | -      | **Required** The unique identifier of the SAP Concur user. Use [Identity v4 API](/api-reference/profile/v4.identity.html) to retrieve the `userID`. |
+| `reportId` | `string` | -      | **Required** The unique identifier of the report that is being read.                                                                                |
+
+### Headers
+
+* [RFC 7231 Accept-Language](https://tools.ietf.org/html/rfc7231#section-5.3.5)
+* [RFC 7231 Content-Type](https://tools.ietf.org/html/rfc7231#section-3.1.1.5)
+* [RFC 7231 Content-Encoding](https://tools.ietf.org/html/rfc7231#section-3.1.2.2)
+* [RFC 7234 Cache-Control](https://tools.ietf.org/html/rfc7234#section-5.2)
+* [RFC 7232 If-Modified-Since](https://tools.ietf.org/html/rfc7232#section-3.3)
+* [RFC 7231 Accept-Encoding](https://tools.ietf.org/html/rfc7231#section-5.3.4)
+* [RFC 7235 Authorization](https://tools.ietf.org/html/rfc7235#section-4.2) - Bearer Token that identifies the caller.
+  This is the Company or User access token.
+* `Concur-CorrelationId` is a Concur specific custom header used for technical support in the form of
```

Comment:
We are not exposing `Concur-CorrelationId`. It is considered as internal only param. Remove.